lib/src/flutter/runners/gherkin_integration_test_runner.dart

```diff
@@ -302,7 +302,7 @@ abstract class GherkinIntegrationTestRunner {
       result = new StepResult(
           0, StepExecutionResult.skipped, "Previous step(s) failed.");
     } else {
-      for (int i = 0; i < this.configuration.stepMaxRetries; i++) {
+      for (int i = 0; i < this.configuration.stepMaxRetries + 1; i++) {
```

Comment:
Probably better to do a Math.max here `max(this.configuration.stepMaxRetries, 1)`

Reply:
But retry would implicate that it would try once, and then try again for x amount of times. We should then rename it to maxTries?